airy.go

```diff
@@ -0,0 +1,19 @@
+package mathext
+
+import "github.com/gonum/mathext/internal/amos"
+
+// Airy returns the Airy function at z.
```

Comment:
This doc comment and below should probably have a more detailed explanation of the function.